### PR TITLE
[Conductor] Add durations to history

### LINKED ISSUE #3783: [Conductor] Add duration to time conductor history.
The current time conductor history entries are difficult to parse. Adding duration might give the user something that can be comprehend quicker than just the start and end bounds.

### PR DESCRIPTION
**Pull request by jvigliotta**
Added time durations to conductor history when in utc time system and in fixed time mode. (didn't do realtime as it seems pretty understandable as is).

Format is **"Starting [start time] for [numeric duration]"**

Example:
Screen Shot 2021-04-16 at 3 51 28 PM

### Author Checklist
Check | Passed
-- | --
Changes address original issue? | Y
Unit tests included and/or updated with changes? | N/A
Command line build passes? | Y
Changes have been smoke-tested? | Y
Testing instructions included? | Y

closes #3783

### CODE REVIEW

File: src/plugins/timeConductor/ConductorHistory.vue [hunk 1]
```diff
@@ -39,6 +39,9 @@ const DEFAULT_DURATION_FORMATTER = 'duration';
 const LOCAL_STORAGE_HISTORY_KEY_FIXED = 'tcHistory';
 const LOCAL_STORAGE_HISTORY_KEY_REALTIME = 'tcHistoryRealtime';
 const DEFAULT_RECORDS = 10;
+const ONE_MINUTE = 60 * 1000;
+const ONE_HOUR = ONE_MINUTE * 60;
+const TWENTYFOUR_HOURS = ONE_HOUR * 24;
```
Comment (nikhilmandlik) [inline src/plugins/timeConductor/ConductorHistory.vue:44]: TWENTY_FOUR_HOURS 😃

File: src/plugins/timeConductor/ConductorHistory.vue [hunk 2]
```diff
@@ -163,6 +176,23 @@ export default {
                 };
             });
         },
+        getDuration(numericDuration) {
+            let result;
+            let age;
+
+            if (numericDuration > TWENTYFOUR_HOURS) {
+                age = (numericDuration / TWENTYFOUR_HOURS).toFixed(2);
+                result = ` for ${age} days`;
```
Comment (nikhilmandlik) [inline src/plugins/timeConductor/ConductorHistory.vue:185]: remove space before ` for....` as its already in `${startTime} ${this.getDuration....`

File: src/plugins/timeConductor/ConductorHistory.vue (hunk 1 above)
Comment (jvigliotta) [reply src/plugins/timeConductor/ConductorHistory.vue:44]: haha, good point, you know, I think I saw that, then didn't do anything about it...

File: src/plugins/timeConductor/ConductorHistory.vue [hunk 3]
```diff
@@ -135,10 +138,20 @@ export default {
     methods: {
         getHistoryMenuItems() {
             const history = this.historyForCurrentTimeSystem.map(timespan => {
```
Comment (nikhilmandlik) [inline src/plugins/timeConductor/ConductorHistory.vue:140]: I do see some duplicate values, may be we can clean up that in this or create new issue.

can convert `historyForCurrentTimeSystem` to Array `Set `

Comment (jvigliotta) [reply src/plugins/timeConductor/ConductorHistory.vue:140]: good call, might as well do it in this one since I'm in that exact code...

**Comment by shefalijoshi** [comment]
@jvigliotta I noticed that every time the time is changed in the time conductor, the history gets updated twice. Any idea why? Can we optimize this to remove the extra update?
image

**Review by shefalijoshi — COMMENTED** [review]
Please see comment about duplicate updates in history.

**Comment by akhenry** [comment]
This looks awesome!

I'd love to get rid of the "starting" somehow, it's just visual noise. A columnar layout would probably be better (but would be a lot of work). Failing that could we just do something like "2021-04-24 16:06:39:334Z + 2.02 days"?

**Review by shefalijoshi — APPROVED** [review]
### Reviewer Checklist

1. Changes appear to address issue? Y
2. Appropriate unit tests included? N
3. Code style and in-line documentation are appropriate? Y
4. Commit messages meet standards? Y
5. Has associated issue been labelled `unverified`? (only applicable if this PR closes the issue) Y

**Pull request merged**